client/packages/system/src/Name/api/api.ts

```diff
@@ -160,4 +161,14 @@ export const getNameQueries = (sdk: Sdk, storeId: string) => ({
       throw new Error('Unable to fetch properties');
     },
   },
+  updateNameProperties: async (input: UpdateNamePropertiesInput) => {
+    const result = await sdk.updateNameProperties({ storeId, input });
+
+    if (result.updateNameProperties.__typename === 'NameNode') {
+      return result?.updateNameProperties;
+    }
+
+    // TODO: properly handle structured error
```

Comment:
Only structured error is NameNotFound right? Shouldn't really happen in the UI?

Reply:
Yeah arguably could change to be a standard error - I think "not found" was pushed to be a structured error for the use case where while you are editing, someone else deletes the name, but that seems more likely for something like invoice line than it does for name 😅